docs/tutorials/shallow-join.rst

```diff
@@ -0,0 +1,359 @@
+Visualizing Analytics in Jupyter
```

Comment:
We may want to reconsider our terminology, @jnh5y.  "shallow join" appears to be a new term for the previously well established practice of aggregating over an inner join (albeit, a join using a geospatial predicate, which again is not novel).  Perhaps, "jupyter-aggregation" is a better name for this tutorial?